feature-utils/file-storage/src/index.js

```diff
@@ -1,4 +1,4 @@
-export default class Storage {
+export class FeatureFileStorage {
```

Comment:
I'm *very* reluctant to
1. Mix changes to API in something that is basically a feature PR.
2. Include features in the API without tests.

In this case, well, it didn't have unit tests to start with, and basically what's being done is to move it from its initial fbImport place. Still, this means that this PR is going to include a set of changes related to this removal *and* another that's focused in what this is about, setting up the requested feature.

If at all possible, I would tackle them separately. Clearly having this is a blocker for importing the feature, so if you want we can do it fast-ish (I can help you with tests if needed), and open a separate PR just for this, which we can merge before coming back here. So what I propose is

1. Creating a branch from (current) main.
2. Check out all file-storage files and resulting changes in fbImport
3. Add tests
4. Create a PR and try to deal with that *pronto*

Reply:
I get your point with testing that is fait, yet as there were no tests prior this could be done in a separate PR following this one (we will encounter a lot of code in the future importer work that is untested and cannot start delaying all PR due to this IMHO). What I do not fully understand is this "Mix changes to API in something that is basically a feature PR.": This is not an API change! Storage works with the API but is a non-API feature utility.

Reply:
Edit: There are quite some tests for importer to be fair :)

Reply:
Well, it's moving something to the current API. My point is that changes to features, or creation of them, should be separated to changes in the API, or additions to them. I expressed it in this ticket https://jira.polypoly.eu/browse/PROD4POD-957, because it's something that's done quite commonly.
In this specific case, we will need (probably later) factoring off some stuff in fbImport to poly-look, for instance. It's probably a bit late to do separate PRs for this one (and you might not want to anyway), but I would *really* beg you to try and do it in the future...

Reply:
I totally get your point about separating these and it makes absolute sense. Still I don't get in what regard you mean changes to the API. Is silly-i18n part of the API? I do not think so. Same with this as we move it to feature-utils not feature-api or am I mistaking completely here?

Reply:
My point is to separate changes to features to changes that are outside the boundary of the feature we're changing. silly-i18n is not part of the polypod API, it's literally as you say feature-utils. Still, I would expect someone needing some functionality from silly-i18n to create first a PR for it (or dummy-server, for that matter (come to think of it, it looks like anything that goes to feature-utils needs a deprecatory prefix, so you'd better think about something for storage :-) )) and then go ahead and introduce it in whatever feature, once it's been well tested, accepted and merged. 
We've put a lot of thought into directory layout for feature-utils and feature-api, so we should expect the same amount of due diligence when including something else
> In that sense, storage could be considered part of the polypod API, not the utils. Going for one instead of the other is precisely the call that could be better considered in its own PR.

Reply:
What we are doing now is quite similar to what we are doing in poly-look. We are reusing code from fbImport for google. Storage in that sense is not part of the API but a utility to elegantly work with the API, which is quite different than the API itself. IMHO importers and feature storage are prime candidates for being utilities on a feature level.
Working with something that is that closely related to features makes it quite hard to work with "on it's own" as you would always need to change things in the feature in the same go.

Reply:
I'm not saying it's not. I'm just saying that by giving it its own PR, we can argue in one direction or another and reach a consensus.

Reply:
Ok will add a PR off main with just that.